public/data/KendallDoesCoding.json

```
@@ -2,11 +2,21 @@
   "name": "Kendall Pinto",
   "type": "personal",
   "bio": "Hello, My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, JavaScript, and the one I'm currently working on projects with, HTML & CSS. I'm currently working on the project, A Very Mogul Christmas",
-  "avatar": "https://i.ibb.co/5LsjwVB/Avatar-Maker.png",
+  "avatar": "https://github.com/KendallDoesCoding.png",
   "links": [
     {
       "name": "Check out my personal website",
-      "url": "https://kendalldoescoding.gq/",
+      "url": "https://kendalldoescoding.gq",
+      "icon": "globe"
+    },
+    {
+      "name": "Coding Youtube",
```

I think you should capitalize the `t` 😅 What do you think?
```suggestion
      "name": "Coding YouTube",
```

---

yup lol